Bibliotheque nationale de France.js

```diff
@@ -400,10 +417,28 @@ var BnfClass = function() {
 		//Fix creators
 		getCreators(record, newItem);
 
-		//Store perennial url from 009 as attachment and accession number
+		
+		//Fix callNumber
+		getCote(record, newItem) ;
+		 
+		//Store perennial url from 003 as attachment and accession number
+		
 		var url = record.getField("003");
 		if(url && url.length > 0 && url[0][1]) {
-			newItem.url = url[0][1];
+			newItem.attachments.push({
+			title: 'Lien vers la notice du catalogue',
+			url: url[0][1],
+			mimeType: 'text/html',
+			snapshot: false
+		});
+		//also add snapshot using permalink so that right-click -> View Online works
```

Comment:
right-click --> view online also works with the link attached above, so this really isn't necessary and saving snapshots of library catalog entries seems excessive to me. Please delete this part.